### ISSUE TITLE
Run as Non Root not possible anymore

### ISSUE THREAD

**Issue by setiseta** [docker-library/tomcat#68]
Hello,

since the change to tomcat 8.5 i can not start it as non root.
i think this is not wanted. so i post this issue.
File permissions now:
-rwxr-x---  1 root root   22K Mar 27 14:25 catalina.sh

File Permissions on older images:
-rwxr-xr-x  1 root root   22K Mar 28 14:44 catalina.sh

Kind Regards
Sebastian

**Comment by tianon** [comment]
I can confirm this. :disappointed:

Any idea what happened?  Nothing in the `Dockerfile` itself changed (besides the version number of Tomcat), so perhaps something in the upstream release tarballs changed?

**Comment by setiseta** [comment]
yes i think it changed in the tarball. Maybe set a chmod over the installation dir?
Or report to the upstream?

Kind Regards,
Sebastian

**Comment by tianon** [comment]
Reporting it to upstream would be great -- if they reply, we can know
whether it was intentional or accidental. 👍

**Comment by davidmichaelkarr** [comment]
Would it perhaps be better to add the following to a Dockerfile based on this image?:
`RUN addgroup tomcatusers && adduser -D tomcat && adduser tomcat tomcatusers
RUN chown -R tomcat:tomcatusers .
USER tomcat`
